apps/SageAccountWeb/src/style/_AccountSetting.scss

```diff
@@ -1,13 +1,26 @@
+.account-settings-page {
+  height: 100vh;
+  overflow-y: scroll;
+  position: relative;
+}
```

Comment:
"stick" nav-panel to this ancestor, rather than .App (which has overflow: hidden), see more info [here](https://developer.mozilla.org/en-US/docs/Web/CSS/position#sticky) and [here](https://stackoverflow.com/a/44929597)